Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -376,43 +441,28 @@ Move files:
    :file:`public/typo3temp/var`, unless you need to keep the log files
    or anything else that may still be relevant.
 
-These locations have changed:
-
-+--------------------------------+-----------------------+
-| Before                         | After                 |
-+================================+=======================+
-| :file:`public/typo3conf/sites` | :file:`config/sites`  |
-+--------------------------------+-----------------------+
-| :file:`public/typo3temp/var`   | :file:`var`           |
-+--------------------------------+-----------------------+
-| :file:`public/typo3conf/l10n`  | :file:`var/labels`    |
-+--------------------------------+-----------------------+
-
+These locations have changed, note that TYPO3 v12+ moved some more configuration
+files to a new directory than TYPO3 v11:
+
++------------------------------------------------------+--------------------------------------+
+| Before                                               | After                                |
++======================================================+======================================+
+| :file:`public/typo3conf/sites`                       | :file:`config/sites`                 |
++------------------------------------------------------+--------------------------------------+
+| :file:`public/typo3temp/var`                         | :file:`var`                          |
++------------------------------------------------------+--------------------------------------+
+| :file:`public/typo3conf/l10n`                        | :file:`var/labels`                   |
++------------------------------------------------------+--------------------------------------+
+| :file:`public/typo3conf/LocalConfiguration.php`      | :file:`config/system/settings.php`   |
```

Comment:
Not true, when I am correct: Legacy path should be public/typo3conf/system/settings.php, same with additional.php below.